src/guide/5-running-a-sim/4-collecting-data.rst

```diff
@@ -1,2 +1,42 @@
 Collecting Data
 ===============
+
+Saving Agent State
+------------------
+
+The state of the simulation can be periodically saved in either XML or JSON format. To save the state, supply the **TODO: which switch** switch 
+to the program, followed by the output period. For example
+
+**TODO: Example**
+
+runs program '...' and will output its state every 5 steps in the JSON format.
+
+Collecting Other Information
+----------------------------
```

Comment:
Logging is intended for custom collection, not step function printf. I wouldn't even bother telling people about the ability to print to console that's core language stuff.

Reply:
https://github.com/FLAMEGPU/FLAMEGPU2/blob/master/include/flamegpu/sim/LoggingConfig.h